test-requirements.txt

```diff
@@ -1,4 +1,4 @@
-black==23.3.0
+black==24.3.0
```

Comment:
Neither black nor flake8 is required in test-requirements.txt since the zuul tox linters job was removed. Please have a look at https://github.com/ansible-collections/cisco.nxos/blob/main/test-requirements.txt